react-backend/client/src/components/visitors/qrcode.js

```diff
@@ -16,7 +16,7 @@ const StyledNav = styled.nav`
 `;
 
 const StyledSection = styled.section`
-  margin: 3rem 0;
+  margin: {props => props.margin 'rem 0'}
```

Comment:
Isn't this a syntax error?

Reply:
Na, it sticks the number passed in from props in front of the string.

Reply:
Nope, definitely a syntax error.

<img width="232" alt="screen shot 2018-04-02 at 14 19 14" src="https://user-images.githubusercontent.com/12719553/38197586-eaa9929c-3680-11e8-903e-175dc40c37b9.png">

It's not crashing because you're not doing string interpolation properly. You're missing the `$` in front of the `{`.

Reply:
Oops